src/components/ui/tooltip.tsx

```diff
@@ -46,9 +47,24 @@ const TooltipContent = React.forwardRef<
 ));
 TooltipContent.displayName = TooltipPrimitive.Content.displayName;
 
+const TableCellWithTooltip = ({
```

Comment:
@henri-ly Not sure where to put this... I don't know if components in /src/components/ui should remain "pure" ?
Let me know if that's ok or if it should go somewhere else, thanks.

Reply:
Hmmm we can leave it here. It shouldn't be a pain to move it